Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics_test.py

```diff
@@ -523,3 +523,36 @@ def test_table_with_invalid_name(function: Callable, message: str) -> None:
     with pytest.raises(DemistoException, match=message):
         # in the run_search_job_command the args is the first argument instead of the second because it use the polling decorator
         function({'table_name': 'invalid'}, {'table_name': 'invalid'})
+
+
+@pytest.mark.parametrize("azure_cloud", [(AZURE_WORLDWIDE_CLOUD), (AZURE_US_GCC_CLOUD), (AZURE_US_GCC_HIGH_CLOUD)])
```

Comment:
This was failing before your current changes if the test was existing?

Reply:
Yes.